CHANGES

```diff
@@ -150,7 +150,7 @@ Fixes
 
 DeprecationWarning
   - The default MBAR error estimator in workflow.ABFE.estimate will change from
-    analytic to bootstrap=50 in 2.2.0 (issue #320, PR#322).
+    analytic to bootstrap=50 in 2.6.0 (issue #320, PR#322).
```

Comment:
retroactive change in CHANGES – we didn't change the behavior in 2.2.0 as advertised. I am now just saying we're doing this in 2.6.0 unless someone wants to quickly take on making the change _and_ updating the tests accordingly.